packages/fxa-settings/src/components/CardHeader/index.tsx

```diff
@@ -23,10 +23,16 @@ interface CardHeaderBasicProps extends CardHeaderRequiredProps {
   headingTextFtlId: string;
 }
 
+interface CardHeaderWithCustomSubheadingProps extends CardHeaderRequiredProps {
```

Comment:
Elegant!

Reply:
Props to @LZoog, I'm just following her pattern 🌟